composer.json

```diff
@@ -12,10 +12,17 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^2.3"
+        "squizlabs/php_codesniffer": "^2.3",
+        "escapestudios/symfony2-coding-standard": "~2.0"
+    },
+    "suggest": {
+        "fabpot/php-cs-fixer": "^1.11 - For easy CS fixing"
```

Comment:
@shrikeh 

Alternatively, `squizlabs/php_codesniffer` ships with `phpcbf`, see https://github.com/squizlabs/PHP_CodeSniffer/wiki/Fixing-Errors-Automatically#using-the-php-code-beautifier-and-fixer.